todo_app/app.py

```diff
@@ -1,15 +1,45 @@
-from flask import Flask
-
+from flask import Flask,render_template, request, redirect
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item, save_item, get_item
```

Comment:
This line of imports isn't used. In fact you can get rid of `session_items.py` completely